libethdrivers/src/plat/imx6/imx6.c

```diff
@@ -461,18 +464,21 @@ static void handle_irq(struct eth_driver *driver, int irq)
     assert(enet);
 
     uint32_t e = enet_clr_events(enet, IRQ_MASK);
-    if (e & NETIRQ_TXF) {
-        complete_tx(dev);
-    }
-    if (e & NETIRQ_RXF) {
-        complete_rx(dev);
-        fill_rx_bufs(dev);
-    }
-    if (e & NETIRQ_EBERR) {
-        ZF_LOGE("Error: System bus/uDMA");
-        // ethif_print_state(netif_get_eth_driver(netif));
-        assert(0);
-        while (1);
+    while (e & IRQ_MASK) {
```

Comment:
Should we merge this as a potential bug fix in a separate PR? Looping to consume all pending interrupt seems a good general pattern and I can't remember any reason not to do it here.